RDBM/Table.cpp

```diff
@@ -193,18 +211,106 @@ void Table::Show(std::ostream &out)
 		ptr = table[i];
 		ptr->Show(out);
 	}
+	return *this;
 }
 
 
 
-void Table::Delete()
+Table& Table::Delete()
 {
 	table.clear();
+	return *this;
 }
 
-void Table::Set(unsigned int colindex, unsigned int rowindex,std::istream &in)
+Table& Table::Set(unsigned int rowindex, unsigned int colindex, std::istream &in)
 {
 	table[rowindex-1]->Set(colindex,in);
+	return *this;
+}
+
+Table& Table::Set(unsigned int rowindex, unsigned int colindex, std::string value)
+{
+	table[rowindex - 1]->Set(colindex, value);
+	return *this;
+}
+
+Table& Table::Set(unsigned int rowindex, std::string column, std::string value)
+{
+	rowindex--;
```

Comment:
Add check here. If rowindex will be 0, after decrementing it will be equal to max value of unsigned int. I don't think it is expected behavior.